src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPage.container.js

```diff
@@ -21,8 +23,11 @@ export const EnrollmentPage: ComponentType<{||}> = () => {
     [
         enrollments,
         enrollmentId,
+        teiId,
```

Comment:
Related: When I supply only the enrollmentId in the url; I can see that data from the enrollment/tei endpoints are requested twice. Is this necessary (something I don't grasp here)?

Reply:
This shouldnt be the case. Let me double check please and I will come back to you

Reply:
Things change quite a bit when I realised that we also have to fetch data just by using the `teiId`. So most likely this is been taken care of in one of the next PRs. However I will check and let you know for sure!